src/Microsoft.VisualStudio.ProjectSystem.Managed/ValueTaskExtension.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if !MICROSOFT_CODEANALYSIS_CONTRACTS_NO_VALUE_TASK
+
+#nullable enable
+#pragma warning disable VSTHRD200 // Use "Async" suffix for async methods
+
+namespace System.Threading.Tasks;
+
+internal static partial class RoslynValueTaskExtensions
+{
+#if NET // binary compatibility
+    public static ValueTask<T> FromResult<T>(T result)
+        => ValueTask.FromResult(result);
+
+    public static ValueTask CompletedTask
+        => ValueTask.CompletedTask;
```

Comment:
Is this needed? Are we likely to be calling code compiled for one TF from code that was compiled against another? I don't think so.